opentelemetry-sdk/src/opentelemetry/sdk/environment_variables.py

```diff
@@ -633,3 +633,12 @@
 
 The :envvar:`OTEL_EXPORTER_OTLP_METRICS_CLIENT_CERTIFICATE` is the client certificate/chain trust for clients private key to use in mTLS communication in PEM format.
 """
+
+_OTEL_RESOURCE_DETECTORS = "_OTEL_RESOURCE_DETECTORS"
```

Comment:
do we have any such ENV variables as now?
I remember specification SIG has not accepted yet @srikanthccv' issue to introduce this env variable ([ISSUE](https://github.com/open-telemetry/opentelemetry-specification/issues/2948))

Reply:
Sorry, I should have added more context in the PR description, I'll answer your question there. :v: